SynthShop.Core/Services/Impl/AuthService.cs

```diff
@@ -71,6 +86,7 @@ public async Task<AuthenticationResult> RefreshTokenAsync(string token, Guid ref
 
             if (!ValidateAccessToken(validatedToken))
             {
+                _logger.Warning("Token validation failed for refresh token {RefreshToken}", refreshToken);
```

Comment:
Please remove this log, generally not recommended to log refresh tokens or any other sensitive information such as access tokens, passwords, or personal identifiable information (PII). Logging such sensitive data can lead to security vulnerabilities